selenium/src/test/java/au/org/emii/portal/utils/SeleniumUtil.java

```diff
@@ -0,0 +1,135 @@
+package au.org.emii.portal.utils;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
+import org.apache.http.conn.ssl.SSLContexts;
+import org.apache.http.conn.ssl.TrustStrategy;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import javax.net.ssl.SSLContext;
+import java.io.BufferedReader;
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SeleniumUtil {
+
+    private static Logger log = Logger.getLogger(SeleniumUtil.class.getName());
+
+    public static String executeCurl(String command) {
+        Process curlProc;
+        String curlResponse = null;
+        try {
+            curlProc = Runtime.getRuntime().exec(command);
```

Comment:
I'd use native java methods to test links rather than calling an operating system command such as curl